src/Commands/RestParametersCommand.ts

```diff
@@ -0,0 +1,48 @@
+import { Command } from "./Command";
```

Comment:
Please remove this file and other code changes. Once the design doc PR is finished, we'll do implementation.